vendor/brave-ios/Ads/Generated/NativeAdsClient.mm

```diff
@@ -91,16 +91,16 @@
   [bridge_ URLRequest:url headers:headers content:content contentType:content_type method:method callback:callback];
 }
 
-void NativeAdsClient::Save(const std::string & name, const std::string & value, ads::ResultCallback callback) {
-  [bridge_ save:name value:value callback:callback];
+void NativeAdsClient::Save(const std::string & path, const std::string & value, ads::ResultCallback callback) {
+  [bridge_ save:path value:value callback:callback];
```

Comment:
There are further implications of passing a path vs a name and will need adjustments to iOS client code.

What is the goal here? To store a file at `{storage}/{folder}/{data}` instead of `{storage}/{data}`?

Reply:
@tmancey that originated from your commit I believe

Reply:
@kylehickinson We need the ability to load files from a path, due to the new component updater code that @moritzhaller is working on. And for persisting rewards logs

Reply:
How is that going to work with iOS that doesn't use component updater?

Reply:
We are adding a component updater component like the one used for NTP for iOS.